appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.61.0",
+    "version": "0.61.1",
```

Comment:
Fyi I actually already released a version 0.61.1 today. If you rebase/merge this branch with master you'll see that this doesn't actually change the version

Reply:
Bumped to 0.61.2